File: C99/1000-1099/1010-1019/1015.c
#include <stdio.h>
#include <math.h>

int main()
{
    double n1, n2, n3, n4;
    scanf("%lf %lf %lf %lf", &n1, &n2, &n3, &n4);
    printf("%.4lf\n", sqrt(pow((n3 - n1), 2) + pow((n4 - n2), 2)));
    return 0;
}

File: beecrowd/1000-1099/1010-1019/1015.py
import math

entry1, entry2 = list(input().split()), list(input().split())
entry1, entry2 = [eval(i) for i in entry1], [eval(i) for i in entry2]
total = math.sqrt((entry2[0] - entry1[0])**2 + (entry2[1] - entry1[1])**2)
print(f"{total:.4f}")